conda/recipes/ucxx/recipe.yaml

```
@@ -401,7 +401,7 @@ outputs:
       version: ${{ version }}
     build:
       noarch: python
-      string: ${{ date_string }}_${{ head_rev }}
+      string: py${{ python | version_to_buildstring }}_${{ date_string }}_${{ head_rev }}
```

@jakirkham This is `noarch: python` so presumably we don't want a Python version in the buildstring?

---

Good point!

Was basing this off of [previous builds]( https://anaconda.org/rapidsai-nightly/distributed-ucxx/files?sort=time&sort_order=asc ), which did have the Python version (for [example]( https://anaconda.org/rapidsai-nightly/distributed-ucxx/files/modal/info/67fca0e65729148dffdf803b ))

Looks like `noarch: python` was added as part of the migration to `rattler-build` in PR: https://github.com/rapidsai/ucxx/pull/374

We can simplify this to `py_`, which is typical for `noarch: python` packages

```suggestion
      string: py_${{ date_string }}_${{ head_rev }}
```